src/pages/Admin/Charity/Banking/Banking.tsx

```diff
@@ -28,9 +28,9 @@ export default function Banking() {
 
       const { id, details, currency } = recipient;
       //creating account return V1Recipient and doesn't have longAccount summary field
-      const bankSummary = `${currency.toUpperCase()} account ending in ${details.accountNumber?.slice(
-        -4 || "0000"
-      )} `;
+      const bankSummary = `${currency.toUpperCase()} account ending in ${
+        details.accountNumber?.slice(-4) || "0000"
```

Comment:
@ap-justin was this here by mistake?
Seemed to be that the intention was that `"0000"` was to be used in case the previous value is falsy.

Reply:
right should be outside